include/pulsar/KeySharedPolicy.h

```diff
@@ -98,6 +98,12 @@ class PULSAR_PUBLIC KeySharedPolicy {
      */
     KeySharedPolicy& setStickyRanges(std::initializer_list<StickyRange> ranges);
```

Comment:
The method that accepts an `initializer_list` is not a good API. I think we should mark it as deprecated.

Reply:
Done.